.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+Code of Conduct
+---------------
+
+At webpack and all webpack/webpack-contrib repositories we follow the
+[JSFoundation Code of Conduct][1]. Please adhere to the guidelines there and
+feel free to report any violation of them to the @webpack/core-team or
+@webpack/documentation-team.
```

Comment:
Good call, is there some generic support email address we have or should we just throw some other emails addresses in?